pkg/service/v1alpha1/controller.go

```diff
@@ -70,14 +70,16 @@ func (cs *controller) CreateVolume(
 	volName := req.GetName()
 	size := req.GetCapacityRange().RequiredBytes
 	configClass := req.GetParameters()["configClass"]
+	rCount := req.GetParameters()["replicaCount"]
+	cspcName := req.GetParameters()["storagePoolClaim"]
```

Comment:
this need to be renamed similar to label on CVC like `CStorPoolClusterName`?

Reply:
done !